examples/superposition-demo-app/src/templates/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="/static/htmx.js"></script>
```

Comment:
why are we simply not using cdn provided by the packages, here ?
its anyways a network call over here, so why not fetch the original one, why make copies ?

Reply:
You can work on the project without internet 😉 @ayushjain17